.travis.yml

```diff
@@ -13,11 +9,12 @@ php:
     - 7.3
 
 before_install:
-    - composer require php-coveralls/php-coveralls --prefer-dist --no-interaction --no-progress
-    - composer global require localheinz/composer-normalize --prefer-dist --no-interaction --no-progress
+    - composer require php-coveralls/php-coveralls
+    - composer global require localheinz/composer-normalize --no-interaction --prefer-dist --no-progress
+    - composer self-update --stable
```

Comment:
That's the wrong order, we should update composer at first.
Also: why were all the parameters removed?